data/helpers.d/filesystem

```diff
@@ -80,3 +80,31 @@ properly with chmod/chown." >&2
     chmod 755 $TMP_DIR
     echo $TMP_DIR
 }
+
+# Remove a file or a directory securely
+#
+# usage: ynh_secure_remove path_to_remove
+# | arg: path_to_remove - File or directory to remove
+ynh_secure_remove () {
+	path_to_remove=$1
```

Comment:
We were discussing the usage of wildcards (`*`) with ynh_secure_remove, for instance : `ynh_secure_remove /var/www/$OOPS/*` with `$OOPS` and empty string. This will be expanded into `ynh_secure_remove /var/www/app1 /var/www/app2 /var/www/app3` which for now is a legit request from the point of view of `ynh_secure_remove`.

So the proposal was to throw an error if $2 is not empty (we cannot check that the helpers was called with a * in the path or not :s), and also to check in the App linter if `*` is used on the same line than ynh_secure_remove. Does that sounds reasonable ? We could also merge the PR as it is right now, and discuss this particular point in a separate thread/PR if you guys prefer.